test/classes/initializers/records/generics/reuse-init-instantiation-var.chpl

```diff
@@ -1,17 +0,0 @@
-// This test exercises calling the same initializer twice with the same
-// arguments when a record has a generic var field
-
-record Foo {
-  var x;
-
-  proc init(x) where !isSubtype(x.type, Foo) {
-    this.x = x;
-  }
-}
-
-var foo1: Foo(int);
-var foo2: Foo(int);
```

Comment:
This actually isn't identical to the other test - it uses the same instantiation twice, while the other test uses different instantiations

Reply:
so true. I must have missed up my diff command. I will add it back in.